src/common/responsiveStyleHelperClasses.js

```diff
@@ -2,10 +2,17 @@
 import css from 'styled-jsx/css';
 
 const responsiveStyleHelperClasses = css`
+  .mobileOnly {
+    display: none;
+  }
+
   @media only screen and (min-width: 320px) and (max-width: 480px) {
     .desktopOnly {
       display: none !important;
     }
+    .mobileOnly {
+      display: initial !important;
```

Comment:
This is good, but we have to be careful with `display: initial`, as it might not have the intended effect. (For example it would convert a `div` with `display: flex` into `display: block`, no? ) Unfortunately I cannot think of a nice solution.

Reply:
fixed